File: Python/1036 - Fórmula de Bhaskara.py
import math

a, b, c = map(float, input().split())

delta = b**2 - 4 * a * c

if a == 0 or delta < 0:
    print("Impossivel calcular")
else:
    r1 = (-b + math.sqrt(delta)) / (2 * a)
    r2 = (-b - math.sqrt(delta)) / (2 * a)
    print(f"R1 = {r1:.5f}")
    print(f"R2 = {r2:.5f}")

File: JavaScript/1036 - Fórmula de Bhaskara.js
var [A, B, C] = require('fs').readFileSync('/dev/stdin', 'utf8').split(' ').map(parseFloat);
var delta = B * B - 4 * A * C;

if (A === 0 || delta < 0) {
    console.log("Impossivel calcular");
} else {
    let r1 = (-B + Math.sqrt(delta)) / (2 * A);
    let r2 = (-B - Math.sqrt(delta)) / (2 * A);
    console.log(`R1 = ${r1.toFixed(5)}`);
    console.log(`R2 = ${r2.toFixed(5)}`);
}